vtex/utils/vtexId.ts

```diff
@@ -22,8 +22,8 @@ export const parseCookie = (headers: Headers, account: string) => {
 
   return {
     cookie: stringify({
-      [NAME]: cookies[NAME],
-      [`${NAME}_${account}`]: cookies[`${NAME}_${account}`],
+      ...(cookies[NAME] && {NAME: cookies[NAME]}),
```

Comment:
I think that's not what you meant, right? otherwise a `NAME` key will be created
```suggestion
      ...(cookies[NAME] && {[NAME]: cookies[NAME]}),
```